src/foam/dao/EasyDAO.js

```diff
@@ -61,22 +61,28 @@ foam.CLASS({
     'foam.dao.RequestResponseClientDAO',
     'foam.dao.SequenceNumberDAO',
     'foam.dao.SyncDAO',
-    'foam.dao.TimingDAO'
+    'foam.dao.TimingDAO',
+    {
+      path: 'foam.dao.JournalType',
+      flags: ['java']
+    },
+    'foam.dao.RoutingJournal',
+    'foam.dao.RoutingJDAO'
   ],
 
-  imports: [ 'document' ],
+  imports: ['document'],
 
   constants: [
     {
       // Aliases for daoType
       name: 'aliases',
-      flags: [ 'js' ],
+      flags: ['js'],
       value: {
-        ARRAY:  'foam.dao.ArrayDAO',
+        ARRAY: 'foam.dao.ArrayDAO',
         CLIENT: 'foam.dao.RequestResponseClientDAO',
-        IDB:    'foam.dao.IDBDAO',
-        LOCAL:  'foam.dao.LocalStorageDAO',
-        MDAO:   'foam.dao.MDAO'
+        IDB: 'foam.dao.IDBDAO',
```

Comment:
Why are you breaking alignment and making the code harder to read?

Reply:
This would be the JS linter too. Will fix.